### PR TITLE
Build application-registry

### PR DESCRIPTION
**Pull request by djkruitbosch**
### Project Abstract

The Application Registry is an on-chain repository of applications available on the EOS Mainnet. It will provide input to wallets and other applications to increase user experience and enhance security, including the associated governance.

### For which grant [level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels) are you applying?

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $50,000, 3 approvals
- [x] **Level 3**:  Unlimited, 4 approvals

### Application Checklist

- [x] The [application template]([xxxx](https://github.com/eosnetworkfoundation/grant-framework/blob/main/applications/application-template.md)) has been copied, renamed (`my_project_name.md`) and updated.
- [x] The application file (`my_project_name.md`) has no spaces in its name, ends with the `.md` extension has been placed in the `applications` directory.
- [x] I have read and understood the:
  - [x] [FAQs](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/faq.md), 
  - [x] [application guidelines](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/grant_guidelines_per_category.md) and 
  - [ ] [announcement guidelines](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/announcement-guidelines.md).
- [x] An EOS address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledge the [terms and conditions](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [ ] This project has NOT been part of a token sale or NFT sale.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/eosnetworkfoundation/grant-milestones#process).

### How Did You Hear About our grants program?

- [ ] Social Media
- [ ] EOSNetwork.com
- [x] Personal Recommendation
- [ ] ENF Fireside Chat
- [ ] Investor/VC
- [ ] Online Search
- [ ] Hackathon
- [ ] Other: _______

### CODE REVIEW

File: applications/application-registry.md
Comment (wanderingbort) [inline applications/application-registry.md:176]: `0b` and `0c` imply that this is about indexing however, `1.` is a duplicate of  Milestone 2: `1.` Please clarify what is to be built in this milestone

File: applications/application-registry.md
Comment (wanderingbort) [inline applications/application-registry.md:151]: To what degree is this estimation affected by the discovery in Milestone 1 (if at all)?

Does the structure of governance and metadata significantly change this timeline/delivery?

File: applications/application-registry.md
Comment (wanderingbort) [inline applications/application-registry.md:167]: To what degree is this estimation affected by the discovery in Milestone 1 and any implementation decisions in Milestone 2?

File: applications/application-registry.md
Comment (wanderingbort) [inline applications/application-registry.md:175]: will this deliverable also include some self-hostable application/container for other 3rd parties that want to index the on-chain data?

If so, will Milestone 1 include discovery on the hosting requirements for that backend service?

File: applications/application-registry.md
Comment (wanderingbort) [inline applications/application-registry.md:146]: Please include a section of this deliverable that outlines how multiple versions of the application data standards can co-exist.

Comment (wanderingbort) [inline applications/application-registry.md:146]: Please include a section of this deliverable that outlines how localizations of application data standards will be handled (if at all)

File: applications/application-registry.md
Comment (wanderingbort) [inline applications/application-registry.md:190]: Are "reports" logged on-chain or off? How will these reports be handled in terms of transparency, anonymity (whistleblowing), etc?

**Review by wanderingbort — CHANGES_REQUESTED** [review]
In general, the timeline and pricing is plausible however there are lots of questions to help clarify the scope of the project and more importantly how evaluators will judge "completeness" of deliverables. 

as this is a multiple month project that can be subdivided into smaller projects naturally, I have a strong preference to making multiple smaller applications rather than one larger omni-application. Each milestone seems to be contingent on design and implementation decisions from the previous milestone which necessarily reduces the accuracy of those estimates.

This is not a request, merely a suggestion.  If the other points are given sufficient clarity I will not hesitate to approve the existing structure.

File: applications/application-registry.md
Comment (djkruitbosch) [reply applications/application-registry.md]: This has been corrected in the latest commit. For some reason this got mixed up when parsing the data from the source file (Google Docs) into the Github format.

File: applications/application-registry.md
Comment (torbenanderson) [reply applications/application-registry.md:146]: For this initial version, the data standards will be maintained in English. Future revisions could include localisations. We recommend raising this as an improvement request via the ENF for collaboration with Rewired after the execution of this initial grant.

Comment (torbenanderson) [reply applications/application-registry.md:146]: The Data Standard will be versioned and maintain backward compatibility with previous versions unless a specific breaking change gets identified or is incompatible with improvements made to the EOS blockchain. Should a fork of the standard be required, the community engagement process will commence to ask for and act on feedback. A final decision on the data standard will be with the 3/5 multisign standard custodians.  

The https://github.com/ConsumerDataStandardsAustralia/standards/issues/266 provides a working example of how community engagement would work using GitHub. FIn addition an improvement request could suggest an EOS contract that builds approvals for data standards on chain.

File: applications/application-registry.md
Comment (torbenanderson) [reply applications/application-registry.md:151]: Our role in milestone 1 is to bring the 4 wallet providers together and achieve consensus on future governance and metadata requirements. As we will be taking the lead on this, we are not expecting any major impacts on subsequent milestones.

File: applications/application-registry.md
Comment (torbenanderson) [reply applications/application-registry.md:167]: Our goal and role will be to achieve consensus on the scope at each milestone, before proceeding to the next. The scope of milestone 3 and beyond are not significantly impacted by metadata, governance, and smart contract delivery within the overall defined scope.

File: applications/application-registry.md
Comment (torbenanderson) [reply applications/application-registry.md:191]: We recommend off-chain for privacy reasons and some kind of case management workflow. If needed we can set this up in a project management tool, and also include a place for logging future improvement and maintenance requests. We are open to other suggestions and this can be tabled during the discovery sessions.

**Comment by djkruitbosch** [comment]
@wanderingbort Thank you so much for your feedback, which we reviewed and provided updates for. Let us know if you have any further questions. If not, then we still need the reviews of @larryk85 , @arhag and @spoonincode.

**Comment by djkruitbosch** [comment]
@wanderingbort Let me know if you received the updates we provided about two weeks ago and if any additional clarity would help you to advance/approve this grant application. Big thanks!

**Comment by wanderingbort** [comment]
@tedcahalleos somehow the other reviewers were dropped from this request.

File: applications/application-registry.md
Comment (wanderingbort) [reply applications/application-registry.md:146]: This should be explicitly added to the proposal and not just as a comment on the PR

Comment (wanderingbort) [reply applications/application-registry.md:146]: This should be mentioned explicitly in the application

**Comment by tedcahalleos** [comment]
@djkruitbosch - please do not remove reviewers from the list.  It will only slow down the approval process.

**Comment by djkruitbosch** [comment]
Hi @wanderingbort , I reflected the comments in the main document now, so this should be reflected.
@tedcahalleos The removal of other approvers happened when I tagged wanderingbort, not sure why the other got removed - so sorry

**Comment by tedcahalleos** [comment]
Closing per email sent.

**Pull request closed without merging**
